packages/compass-connections/src/stores/connections-store-redux.ts

```diff
@@ -2219,7 +2218,10 @@ export const removeAllRecentConnections = (): ConnectionsThunkAction<
       toRemove.map((connection) => {
         dispatch(cleanupConnection(connection.info.id));
         track('Connection Removed', {}, connection.info);
-        return connectionStorage.delete?.({ id: connection.info.id });
+        return (
+          connectionStorage.delete?.({ id: connection.info.id }) ??
+          Promise.resolve()
+        );
```

Comment:
(not asking for a change here) This is where I feel like an `async` (keyword) function is more appropriate than substituting a Promise.resolve. Plus `return-await` is a rule we employed in the driver because when you don't do that you remove functions from async stack traces. 

```js
return await connectionStorage.delete?.({ id: connection.info.id });
```